### ISSUE TITLE
Save != publish

### ISSUE THREAD

**Issue by nuest** [o2r-project/o2r-UI#130]
When creating a new ERC, the "Save" button is actually a publish button. It would be helpful, especially for public links for review (#118) and to avoid authors making their ERCs public unintentionally, if "Save" only updates the metadata, and an extra "Publish" button _with a "are you sure" warning for making the ERC publicly available.

**I think we had it that way in the old UI, right?**

 The "are you sure"-dialog could also point out that you can get in touch with the o2r team to get a publicly shareable link.

**Comment by MarkusKonk** [comment]
No, we never had that. We only distinguished between candidates and and non-candicates (which get the status automatically after updating the metadata). Distinction between unpublished ERCs (non-candidates) and published ERCs (also non-candidates) has not been considered somewhere so far, not even in the metadata. But we could just add a field from the UI-side.

**Comment by nuest** [comment]
For me, a candidate is an unpublished compendium, so sorry for not being clear.

There are no "unpublished ERCs", just "candidates", "candidates with a public link", and "published ERCs".

**Comment by MarkusKonk** [comment]
The point is that candidate at the moment is either true or false, no "true with public link" so far. I will just change the metadata.

**Comment by MarkusKonk** [comment]
But: As I rememver, one of your services changes the candidate state automatically from true to false after updating the metadata, if I remember that correctly.

**Comment by nuest** [comment]
You are correct! https://o2r.info/api/compendium/candidate/

I suggest:

- I update the API to enable metadata updates without publishing
- The "Save" button is still renamed to "Publish", because that is what it does, and we add again the "Are you sure" question; the window for this question can also tell the author, that to delete a compendium, she has to reach out to the o2r team (and give our email)
- When the API is updated, we can add an extra "Save metadata" button

**Comment by MarkusKonk** [comment]
Agreed

**Comment by nuest** [comment]
@MarkusKonk Can you go ahead with the button renaming and adding of the "are you sure" question?

**Comment by MarkusKonk** [comment]
Is the API update done?

**Comment by nuest** [comment]
No, but at the moment "Save" effectively is "Publish" and can be renamed. Then, when the API exists (I don't want to rush that right now) we can add the additional "Save" button.

**Comment by nuest** [comment]
1. [ ] API spezifizieren > https://github.com/o2r-project/api/issues/85
2. [ ] API implementieren > https://github.com/o2r-project/o2r-muncher/issues
3. [ ] UI implementieren (this issue)
